optuna_dashboard/ts/components/DataGrid.tsx

```diff
@@ -239,6 +187,76 @@ function DataGrid<T>(props: {
   )
 }
 
+function DataGridHeaderColumn<T>(props: {
```

Comment:
Could this name be better as this component is a header, but not column? (please correct me if I am wrong)

```suggestion
function DataGridColumnHeader<T>(props: {
```